packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.gql.js

```diff
@@ -0,0 +1,56 @@
+import gql from 'graphql-tag';
+
+export const SET_CUSTOMER_INFORMATION = gql`
+    mutation SetCustomerInformation(
+        $firstname: String!
+        $lastname: String!
+        $email: String!
+    ) {
+        updateCustomer(
+            input: { firstname: $firstname, lastname: $lastname, email: $email }
+        ) {
+            customer {
+                id
+                firstname
+                lastname
+                email
+            }
+        }
+    }
+`;
+
+export const CHANGE_CUSTOMER_PASSWORD = gql`
+    mutation ChangeCustomerPassword(
+        $currentPassword: String!
+        $newPassword: String!
+    ) {
+        changeCustomerPassword(
+            currentPassword: $currentPassword
+            newPassword: $newPassword
+        ) {
```

Comment:
```suggestion
        ) @connection(key: 'changeCustomerPassword') {
```

Apollo cache uses the arguments to a mutation or query as part of the cache keys. When those arguments are "secret" or PII, we should use a connection key so that they aren't cached.